tests/behat/features/templates/tags-individual.feature

```diff
@@ -141,9 +141,9 @@ Feature: test individual tags page
     Then I should see "Join"
     Then I should see "Topics from Ask.CI"
     Then I should see "Users"
-    Then I should see "Grant Scott"
-    Then I should see "mentor"
-    Then I should see "ai"
-    Then I should see "management"
+    Then I should see "Name"
+    Then I should see "student facilitator"
```

Comment:
Looks like there aren't student facilitators on all domains. Maybe just use "Roles" until we have some reliable test Users to look for.